docker/jdk9/x86_64/ubuntu/Dockerfile-openj9

```diff
@@ -20,8 +20,18 @@ RUN apt-get update \
   && apt-get install -qq -y --no-install-recommends \
     autoconf \
     libnuma-dev \
+    g++-4.8 \
+    gcc-4.8 \
 && rm -rf /var/lib/apt/lists/*
 
+# Make sure build uses 4.8
+RUN rm /usr/bin/gcc \
+    && rm /usr/bin/g++ \
+    && rm /usr/bin/c++ \
+    && ln -s /usr/bin/gcc-4.8 /usr/bin/gcc \
+    && ln -s /usr/bin/g++-4.8 /usr/bin/g++ \
+    && ln -s /usr/bin/g++-4.8 /usr/bin/c++
```

Comment:
` ln -s /usr/bin/g++-4.8 /usr/bin/c++` is this line correct? it appears the same as ` ln -s /usr/bin/g++-4.8 /usr/bin/g++ ` `c++` and `g++` is mapped to to the same binary executable?

Reply:
um, yes, i'm not super happy about it either, but I found some commands in the build run c++ and some run g++. I doubted that they are so particular about what they want, so I just made them the same. Does that sound reasonable to you?

Reply:
Ok, thanks for sharing this.